packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -70,17 +60,16 @@ const FeedbackStore = types
       }
     }
 
-    function createSubjectMiddleware () {
-      const subjectMiddleware = autorun(() => {
-        addMiddleware(getRoot(self).subjects, (call, next, abort) => {
-          if (call.name === 'advance') {
-            onSubjectAdvance(call, next, abort)
-          } else {
-            next(call)
-          }
-        })
-      }, { name: 'FeedbackStore Subject Middleware autorun' })
-      addDisposer(self, subjectMiddleware)
+    function afterAttach () {
+      addMiddleware(getRoot(self).subjects, _addMiddleWare)
```

Comment:
Why is feedback adding middleware to the subjects store? This is a bit confusing.

Reply:
The tests mock feedback in isolation, so `getRoot(self).subjects` is undefined when they run.